addons/website_sale/views/snippets/s_products_searchbar.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_products_searchbar_input" name="Products Search Input">
+    <t t-call="website_sale.website_sale_products_search_box">
+        <t t-set="_classes" t-value="'s_wsale_products_searchbar_input mb-3'"/>
+    </t>
+</template>
+<template id="s_products_searchbar" name="Products Search">
+    <section class="s_wsale_products_searchbar bg-200 pt24 pb24">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 offset-lg-2">
+                    <div class="card">
+                        <div class="card-body">
+                            <h2>Search for a product</h2>
+                            <p>We have amazing products in our shop, check them now !</p>
+                            <t t-call="website_sale.s_products_searchbar_input"/>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="snippet_options" inherit_id="website.snippet_options" name="e-commerce snippet options">
```

Comment:
Be careful, you now define two views in website_sale with the same key so this one will override the first one. My guess it all the other options of website_sale are thus gone :/

In fact this one can also probably inherit from the first one instead of `website.snippet_options`